.travis.yml

```diff
@@ -0,0 +1,33 @@
+language: cpp
+python:
+  - "2.7"
+install:
+  # Install ARM GCC
+  - sudo add-apt-repository ppa:team-gcc-arm-embedded/ppa -y
+  - sudo apt-get update
+  - sudo apt-get install gcc-arm-embedded -y
+  # Install yotta
+  - pip install --user yotta
+script:
+  # Build all tests for frdm-k64f-gcc
+  - yotta target frdm-k64f-gcc
```

Comment:
how can we expand the list of platforms? It would be good to have at least mkit-gcc there for now.